src/pages/Home/Home.tsx

```diff
@@ -1,5 +1,8 @@
 import React from 'react';
 import shallow from 'zustand/shallow';
+import { useTranslation } from 'react-i18next';
+
+import '../../styles/accessibility.scss';
```

Comment:
I think this import can be moved to be in the main SCSS file. It currently creates a global rule that we only should define once. We probably want to split the global selector from the module selector:

```tsx
import styles from './Home.module.scss';

return <div className={styles.hideUntilFocus} />
```

VS

```tsx
import '../../styles/accessibility.scss';

return <div className="hideUntilFocus" />
```